dev/create-release/do-release-docker.sh

```diff
@@ -62,7 +62,7 @@ while getopts "d:hj:ns:t:" opt; do
     j) JAVA="$OPTARG" ;;
     s) RELEASE_STEP="$OPTARG" ;;
     h) usage ;;
-    ?) error "Invalid option. Run with -h for help." ;;
+    \?) error "Invalid option. Run with -h for help." ;;
```

Comment:
Do we need to fix line 57 by adding `"d` -> `":d`?

Reply:
In this case, $OPTARG is not referred when it matches "\?" so  it's not necessary for the purpose.
But it might be better to make the parameter starts with ":".
When the parameter starts with ":", getopts work in silent mode which doesn't echo error message when invalid options are passed
This script prints error message in itself so non-silent mode prints error message redundantly.

This is an example.
```
# -x is an invalid option.
$ dev/create-release/do-release-docker.sh -x
dev/create-release/do-release-docker.sh: illegal option -- x  // This message is printed by getopts.
Invalid option. Run with -h for help.
```
Do you think it's better to remove this redundancy?

Reply:
Ya. Thanks. That's what I asked: Hiding `illegal option -- x`.
I prefer the current customized Spark error message.

Reply:
O.K, I'll fix it.

Reply:
Thank you!